app/src/main/res/layout/third_party_dependency_item.xml

```diff
@@ -34,7 +33,8 @@
         android:paddingTop="16dp"
         android:paddingEnd="16dp"
         android:paddingBottom="4dp"
-        android:text="@{viewModel.dependencyName}" />
+        android:textColor="@color/component_color_third_party_dependency_list_activity_items_background_color"
```

Comment:
`component_color_third_party_dependency_list_activity_dependency_name_text_view_color `

Its not `items` , instead its a proper textview.
Also, not background but text color.

Reply:
done